### ISSUE TITLE
Getting exception after upgrading to latest version

### ISSUE THREAD

**Issue by dekelev** [apify/fingerprint-suite#52]
Getting this exception after upgrading to latest version:
```
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at BayesianNode.sample (.../node_modules/generative-bayesian-network/bayesian-node.js:79:39)
    at BayesianNetwork.generateSample (.../node_modules/generative-bayesian-network/bayesian-network.js:41:42)
    at FingerprintGenerator.getFingerprint (.../node_modules/fingerprint-generator/fingerprint-generator.js:35:62)
    ...
```

**To Reproduce**
Upgrade fingerprint-generator from v2.0.1 to v2.0.2

**Expected behavior**
No exception in patch upgrade

**System information:**
 - OS: MacOS
 - Node.js version: v16.14.0

**Comment by barjin** [comment]
Thank you for letting us know! From what I can see, there was a problem with the current ML model - honestly, I don't know how this passed the automated pre-publish tests. 

Anyway - tonight, the models were automatically regenerated (and version 2.0.3 was published), which solved this issue - at least for me. Try installing the newest version and let us know how does it look.

Closing this for now, feel free to reopen if any problems resurface :)

**Comment by dekelev** [comment]
I see now that generative-bayesian-network is not upgrading to latest version together with the fingerprint-generator lib upgrade.

The issue is that when running the following command, generative-bayesian-network lib won't update:
`npm update fingerprint-generator`

The version of generative-bayesian-network lib after the update (should be v2.0.2):
<img width="357" alt="image" src="https://user-images.githubusercontent.com/4444781/182134651-d5e90268-4418-4d3a-8d23-605cd6fc9c1b.png">

**Comment by tzbo** [comment]
> @barjin I see now that generative-bayesian-network is not upgrading to latest version together with the fingerprint-generator lib upgrade.
> 
> The issue is that when running the following command, generative-bayesian-network lib won't update: `npm update fingerprint-generator`
> 
> The version of generative-bayesian-network lib after the update (should be v2.0.2): <img alt="image" width="357" src="https://user-images.githubusercontent.com/4444781/182134651-d5e90268-4418-4d3a-8d23-605cd6fc9c1b.png">
> 
> That's why I'm still getting same errors after upgrading fingerprint-generator to latest version, also with v2.0.3.

Yes, I got the same error.

**Comment by B4nan** [comment]
@dekelev @tzbo Maybe try to wipe your node_modules folder and lock file, that should help. 

@barjin I believe we should depend on latest versions, you have the dependencies between the suite packages just as `^2.0.0`, that's how we handle peer deps, but for regular deps it should point to the latest - as we simply dont guarantee that generator 2.0.2 is compatible with injector 2.0.0, and by specifying e.g. `^2.0.2` you force usage of the compatible version.

**Comment by barjin** [comment]
Sorry about the caused problems @dekelev @tzbo, the versioning should be fixed now (`fingerprint-generator@2.0.4` depends on the correct versions of the other packages, just as @B4nan said). 

Thank you for your patience! :)

**Comment by dekelev** [comment]
Awesome, thanks @barjin
